src/tests/ftest/launch.py

```diff
@@ -953,6 +958,161 @@ def rename_logs(avocado_logs_dir, test_file):
                 test_logs_dir, new_test_logs_dir, error))
 
 
+USE_DEBUGINFO_INSTALL = True
+
+def install_debuginfos():
```

Comment:
I think it would make sense to only call this method once in `main()` before calling `run_tests()`, when `args.process_cores` is set.

Reply:
But the debuginfos are not needed if not core files are generated, which is hopefully most of the time, so we shouldn't waste the time and resources installing them for the minority cases.

Ultimately the debuginfos to install differ based on which binary created the core file, so they could be different for each core file.  This is just an initial implementation which covers daos and python.  A future expansion could include installing the debuginfos needed for the specific binary that generated the core.

Reply:
Ah, thanks for the additional info.